aspnetcore/blazor/components/render-modes.md

```diff
@@ -278,17 +265,17 @@ In the following example, the render mode is set to WebAssembly with `@attribute
 
 If using the preceding component locally in a Blazor Web App, place the component in the client project's `Pages` folder. The client project is the solution's project with a name that ends in `.Client`. When the app is running, navigate to `/render-mode-3` in the browser's address bar.
 
-## Auto render mode
+## Automatic (Auto) render mode
```

Comment:
Can we just stick with auto? I get that it probably violates some doc guideline of not abbreviating stuff, but it's what's shipping in the API.